src/k8s/pkg/k8sd/pki/k8sdqlite.go

```diff
@@ -24,20 +24,20 @@ type K8sDqlitePKIOpts struct {
 	Hostname          string
 	DNSSANs           []string
 	IPSANs            []net.IP
-	Years             int
+	Seconds           int
 	AllowSelfSignedCA bool
 	Datastore         string
 }
 
 func NewK8sDqlitePKI(opts K8sDqlitePKIOpts) *K8sDqlitePKI {
-	if opts.Years == 0 {
-		opts.Years = 1
+	if opts.Seconds == 0 {
+		opts.Seconds = 365 * 24 * 60 * 60
 	}
```

Comment:
OK, i hate to be "that guy" -- but these constants aren't always technically a "year"

Reply:
I know it's wordy but at least it's exactly a year from now

```suggestion
	if opts.Seconds == 0 {
	    now := time.Now()
	    duration := now.AddDate(1, 0, 0).Sub(now)	
	    ops.Seconds = duration.Seconds()
	}
```